etc/helm/pachyderm/values.yaml

```diff
@@ -195,6 +195,7 @@ pachd:
     #apiGrpcPort:
     #  expose: true
     #  port: 30650
+  enterpriseLicenseKey: ""
```

Comment:
You'll want to run `make schema` in `etc/helm` to bring the json schema up to date